tests/Geb/positive/Core/test002.jvc

```diff
@@ -4,4 +4,9 @@ type optbool {
   Nothing : optbool;
 };
 
-(\(x : bool) \(o : optbool) { case o of { Just b := if x then true else b; Nothing := false } }) false (Just true)
+(\(x : bool) \(o : optbool) { 
+case o of { 
+  Just b := if x then true else b; 
+  Nothing := false } }) 
+  false 
+  (Just true)
```

Comment:
This formatting change actually breaks the formatting, because the `case` has the wrong indentation.